concrete-storage/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "storage",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "firebase": "^6.0.4",
```

Comment:
Perhaps we should freeze these versions here and below (remove the '^'), setting the version to whatever is currently installed that we know is last operational.

Reply:
Yes